config/default/core.entity_form_display.node.judgement.default.yml

```diff
@@ -0,0 +1,357 @@
+uuid: d6308cb8-4066-40f5-934b-43d0c2622688
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.judgement.body
+    - field.field.node.judgement.field_case_history
+    - field.field.node.judgement.field_casename
+    - field.field.node.judgement.field_casenumber
+    - field.field.node.judgement.field_cases_considered
+    - field.field.node.judgement.field_court_name
+    - field.field.node.judgement.field_external_link
+    - field.field.node.judgement.field_files
```

Comment:
Re-use existing files field from legislation?

Reply:
Ya, this would probably be possible but I went according to the spreadsheet, there are quite a few comments on the spreadsheet that have not been addressed however, but will all be easily adjustable.

Reply:
Again, I think this *is* re-using it.

Reply:
Yes, you are correct it is reusing.